investigations/summer-fi-exploit-2026-07/hacker-funds-tracker/app/globals.css

```
@@ -0,0 +1,622 @@
+  /* data marks (validated categorical steps — dataviz reference palette) */
+  --held: #2a78d6; /* DAI still in the attacker's wallet (trackable)   */
+  --held-dim: #cfe0f6;
+  --staged: #eda100; /* ETH staged in the cash-out wallet (in transit)  */
+  --staged-dim: #f6e3bd;
+  --gone: #e34948; /* converted / sent to Tornado (unrecoverable)     */
+  --gone-dim: #f2cccc;
+
+  --radius: 18px;
+  --radius-sm: 11px;
+  --shadow: 0 1px 2px rgba(11, 11, 11, 0.04), 0 8px 30px rgba(11, 11, 11, 0.06);
+  --font: system-ui, -apple-system, "Segoe UI", Roboto, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)